### PR TITLE
Update .travis.yml to set up runs for different env versions

### LINKED ISSUE #216: Set up test runs with different RQ versions
Need to test with Travis against some 0.x, 1.0, 1.1 and master.

### PR DESCRIPTION
**Pull request by coolhacker**
Closes: #216

There is no need to explicitly specify dist: xenial - is default travis dist

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+  - 3.4
+  - 3.5
+  - 3.6
+  - 3.7
+  - pypy
+env: 
+  - RQ_VERSION="rq==0.12"
+  - RQ_VERSION="rq==1.0"
+  - RQ_VERSION="rq==1.1"
+  - RQ_VERSION="git+https://github.com/rq/rq.git"
```
Comment (eoranged) [inline .travis.yml]: Please add egg name, so It resolves faster.

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
+  - "bash -c '[[ $(python -V) == \"Python 3.4.\"* ]] && pip install setuptools==24.0.3 || :'"
+  - "pip install -r requirements.txt"
+  - "pip install $RQ_VERSION"
+  - "pip install flake8 pytest"
+script: 
   - pytest
-  - flake8 rq_dashboard tests
+  - "flake8 rq_dashboard tests"
+services: 
+  - redis-server
```
Comment (eoranged) [inline .travis.yml]: Please always add newline to the end of file. You can configure it in your editor :)

File: .travis.yml (hunk 1 above)
Comment (coolhacker) [reply .travis.yml]: Edited.

File: .travis.yml (hunk 2 above)
Comment (coolhacker) [reply .travis.yml]: Yeah, set it up

**Review by eoranged — APPROVED** [review]

**Pull request merged**